core/src/test/resources/json-ld.org/frame-0022-frame.jsonld

```diff
@@ -1,12 +1,4 @@
 {
-  "@context": {
-    "dc": "http://purl.org/dc/elements/1.1/",
-    "ex": "http://example.org/vocab#"
-  },
-  "@type": "ex:Library",
-  "ex:contains": {
-    "@explicit":true,
```

Comment:
My impression was that this test file was intended to be a custom test. However, it seems to have mistakenly landed in ``core/src/test/resources/json-ld.org/`` instead of ``core/src/test/resources/custom``. The PR which included it was #168 by @hmottestad for background reference.

Could you move it to ``core/src/test/resources/custom`` as part of this PR?